modules/launcher/src/main/resources/dbscripts/h2-mb.sql

```diff
@@ -47,7 +47,7 @@ CREATE TABLE IF NOT EXISTS MB_METADATA (
                 EXCHANGE_NAME VARCHAR(256) NOT NULL,
                 ROUTING_KEY VARCHAR(256) NOT NULL,
                 CONTENT_LENGTH BIGINT NOT NULL,
-                MESSAGE_METADATA VARBINARY(65000) NOT NULL,
+                MESSAGE_METADATA VARBINARY(60000) NOT NULL,
```

Comment:
Why did we reduce this?

Reply:
In DTX enqueue metadata value exceeds table row size. Hence reduced the metadata size to accommodate that. The reduced value won't be an issue  IMO